src/style.css

```diff
@@ -1 +1,63 @@
-/* Styles go here */
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  --colorAction: #00acdc;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+.header {
+  background-color: #fff;
```

Comment:
it's by default

Reply:
thanks for the comment, resolved in this commit https://github.com/mate-academy/layout_moyo-header/pull/2452/commits/3f7ee88c36c8f23f9912efb51a309b5e7d588b43